scripts/install-snowcli.sh

```diff
@@ -1,24 +1,20 @@
-﻿#!/bin/sh
+#!/usr/bin/bash
```

Comment:
The script was using bashisms before -- pipefail is one example.

Also BOM was removed from file on this line.